bobtimus/src/lib.rs

```diff
@@ -304,16 +304,7 @@ where
                     interest_mod: Decimal::ZERO,
                 },
             ],
-            collateralizations: vec![
-                Collateralization {
-                    collateralization: dec!(1.5),
-                    interest_mod: Decimal::ZERO,
-                },
-                Collateralization {
-                    collateralization: dec!(2.0),
-                    interest_mod: Decimal::ZERO,
-                },
-            ],
+            initial_ltvs: vec![Ltv { ltv: dec!(0.5) }, Ltv { ltv: dec!(0.7) }],
```

Comment:
I am not sure about the the name `initial_ltv` - are these thresholds for the borrower?

Reply:
OK, given the code in `validate_loan_is_acceptable` this is not an interval, but we check against exact matches.

Most user interfaces I looked at don't let the user chose the LTV but the collateralization rate - and then calculate the LTV. I think with this change the user will have to select the LTV. I think we can do that, just pointing out the implication.